portfolio/src/main/webapp/cook.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Food Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+ <body>
+
+ <div class="foodcontent" >
+     <br/>
+    <h2 class="center">Food Gallery</h2>
+    <img class="pfp center" style="width:250px;height:250px;" src="img/shef.png" />
```

Comment:
Generally, inline styling is not recommended. Is there a reason you can't include this info in the "pfp center" class?

Reply:
Yeah I tried to remove most inline styling but I thought I could get away with it for sizing,  but I changed this and others I found.